documentation/library_guide/pstl/dpcpp_policies_usage.rst

```diff
@@ -181,6 +181,18 @@ requirements:
 * Has a ``get_buffer`` method with no arguments. The method returns the SYCL buffer passed to
   ``oneapi::dpl::begin`` and ``oneapi::dpl::end`` functions
 
+Also ``begin``, ``end`` can optionally accept SYCL deduction tags and ``sycl::no_init``
+(See SYCL 2020: 4.7.6.3, 4.7.6.4 for details) to explicitly mention which access mode should be applied
+to the buffer accessor when submitting DPC++ kernel to a device, e.g.
```

Comment:
```suggestion
``begin``, ``end`` can optionally take SYCL 2020 deduction tags and ``sycl::no_init`` as the arguments to explicitly mention, which access mode should be applied to the buffer accessor when submitting DPC++ kernel to a device. For example:
```

I would not use some paragraph numbers in SYCL spec because they are floating. Unless they have something persistent (like tags in C++ standard) I would just use SYCL 2020.

As far as I can tell _e.g._ is used when you want to continue the current sentence and it currently hard to read it when you have _e.g._ and then just the code. The sentence looks incomplete. _For example_ serves better the same goal in my opinion

Reply:
Agree. Thanks